### ISSUE TITLE
ESCI-ID missing for V classified publications

### ISSUE THREAD

**Issue by mietcls** [ugent-library/biblio-backoffice#842]
# Bug description

Example: record `8768475` does not have an ESCI-ID.
It is needed V classified publications, mostly for reviewers.

Note; it is already there for A2 publications.

![Image](https://user-images.githubusercontent.com/1572885/203262957-a56df352-987f-4a72-aa9a-b107ba72fbfa.png)

## Expected behaviour

Old interface:
![Image](https://user-images.githubusercontent.com/1572885/203263135-316df4e6-eabe-4314-9bd9-ea1ff58c5473.png)

Mock in new interface:
![Image](https://user-images.githubusercontent.com/1572885/203263368-1fe11d72-5688-4a56-a15f-8cb3508b5804.png)

## Suggested implementation

```
<div class="form-group row">
  <label class="col-lg-3 col-form-label" for="esci_id">
    ESCI-ID
  </label>
  <div class="col-lg-3">
    <input class="form-control" name="esci_id" type="text" value="">
    <small class="form-text text-muted">e.g. 000503382400004</small>
  </div>
</div>
```

**Comment by nicolasfranck** [comment]
Problem is a missing map the RIS decoder (WOS) to extract the esci id.

Other records do have esci_id.

**Comment by nics** [comment]
resolved by ff6b28ad5ac408ca415b1bc9ab4db263f20eb954

**Comment by nics** [comment]
> Problem is a missing map the RIS decoder (WOS) to extract the esci id.
> 
> Other records do have esci_id.

@nicolasfranck librecat also didn't have a mapping
